### ISSUE TITLE
change `run_mpas_analysis` to `mpas_analysis` entry point

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#452]
According to @jhkennedy, this is likely a necessary step before we can move over to conda-forge.  It also allows our script to have the same name as the package, which would be nice.

Here's some info on entry points:
https://amir.rachum.com/blog/2017/07/28/python-entry-points/

**Comment by xylar** [comment]
This is also causing issues with the latest Read The Docs, so it would be good for documentation purposes to switch to an entry point.

**Comment by jhkennedy** [comment]
@xylar If you don't mind me taking until early next week to get to it, you can assign this to me.

**Comment by xylar** [comment]
@jhkennedy, It's marked as low priority, so if you want to take this on, that would be great!
